test/e2e/network/loadbalancer.go

```diff
@@ -969,6 +972,138 @@ var _ = common.SIGDescribe("LoadBalancers", func() {
 		ginkgo.By("hitting the TCP service's LoadBalancer")
 		e2eservice.TestReachableHTTP(tcpIngressIP, svcPort, loadBalancerLagTimeout)
 	})
+
+	ginkgo.It("should be able to preserve UDP traffic when server pod cycles for a LoadBalancer service", func() {
+		// requires cloud load-balancer support
+		e2eskipper.SkipUnlessProviderIs("gce", "gke")
+		ns := f.Namespace.Name
+		nodes, err := e2enode.GetBoundedReadySchedulableNodes(cs, 2)
+		framework.ExpectNoError(err)
+		if len(nodes.Items) < 2 {
+			e2eskipper.Skipf(
```

Comment:
Skip, not fail?

Reply:
we don't have a good criteria here, I think that we can justify any action ehre, but I prefer to be gentle and not make the test fail to avoid surprises on consumers of the tests